CSV/49-2/AerodromePresentOrForecastWeather/AerodromePresentOrForecastWeather_entity.csv

```diff
@@ -1,403 +1,403 @@
-http://codes.wmo.int/306/4678/TSSNRA,TSSNRA,stable,Thunderstorm with precipitation of snow and rain,Thunderstorm with precipitation of snow and rain,http://codes.wmo.int/306/4678/TSSNRA
-http://codes.wmo.int/306/4678/TSSNRAGR,TSSNRAGR,stable,"Thunderstorm with precipitation of snow, rain and hail","Thunderstorm with precipitation of snow, rain and hail",http://codes.wmo.int/306/4678/TSSNRAGR
-http://codes.wmo.int/306/4678/TSSNRAGS,TSSNRAGS,stable,"Thunderstorm with precipitation of snow, rain and snow pellets/small hail","Thunderstorm with precipitation of snow, rain and snow pellets/small hail",http://codes.wmo.int/306/4678/TSSNRAGS
-http://codes.wmo.int/306/4678/TSUP,TSUP,stable,Thunderstorm with unidentified precipitation,Thunderstorm with unidentified precipitation,http://codes.wmo.int/306/4678/TSUP
-http://codes.wmo.int/306/4678/UP,UP,stable,Unidentified precipitation,Unidentified precipitation,http://codes.wmo.int/306/4678/UP
-http://codes.wmo.int/306/4678/VA,VA,stable,Volcanic ash,Volcanic ash,http://codes.wmo.int/306/4678/VA
-http://codes.wmo.int/306/4678/VCBLDU,VCBLDU,stable,Blowing dust in the vicinity,Blowing dust in the vicinity,http://codes.wmo.int/306/4678/VCBLDU
-http://codes.wmo.int/306/4678/VCBLSA,VCBLSA,stable,Blowing sand in the vicinity,Blowing sand in the vicinity,http://codes.wmo.int/306/4678/VCBLSA
-http://codes.wmo.int/306/4678/VCBLSN,VCBLSN,stable,Blowing snow in the vicinity,Blowing snow in the vicinity,http://codes.wmo.int/306/4678/VCBLSN
-http://codes.wmo.int/306/4678/VCDS,VCDS,stable,Duststorm in the vicinity,Duststorm in the vicinity,http://codes.wmo.int/306/4678/VCDS
-http://codes.wmo.int/306/4678/VCFC,VCFC,stable,Funnel cloud(s) (tornado or water-spout) in the vicinity,Funnel cloud(s) (tornado or water-spout) in the vicinity,http://codes.wmo.int/306/4678/VCFC
-http://codes.wmo.int/306/4678/VCFG,VCFG,stable,Fog in the vicinity,Fog in the vicinity,http://codes.wmo.int/306/4678/VCFG
-http://codes.wmo.int/306/4678/VCPO,VCPO,stable,Dust/sand whirls (dust devils) in the vicinity,Dust/sand whirls (dust devils) in the vicinity,http://codes.wmo.int/306/4678/VCPO
-http://codes.wmo.int/306/4678/VCSH,VCSH,stable,Shower(s) in the vicinity,Shower(s) in the vicinity,http://codes.wmo.int/306/4678/VCSH
-http://codes.wmo.int/306/4678/VCSS,VCSS,stable,Sandstorm in the vicinity,Sandstorm in the vicinity,http://codes.wmo.int/306/4678/VCSS
-http://codes.wmo.int/306/4678/VCTS,VCTS,stable,Thunderstorm in the vicinity,Thunderstorm in the vicinity,http://codes.wmo.int/306/4678/VCTS
-http://codes.wmo.int/306/4678/VCVA,VCVA,stable,Volcanic ash in the vicinity,Volcanic ash in the vicinity,http://codes.wmo.int/306/4678/VCVA
+id,notation,status,description,label,altLabel,source,seeAlso,related,note,iwxxmVersionInfo
```

Comment:
the related URLs are identical to the id URLs

Reply:
The "related" column has been emptied.